(DOCSP)-29384 Clarify the need to name the custom schema when loading it into drdl (#413)
mongodb/docs-bi-connector: source/schema/load-schema-from-drdl.txt

Reviewer: I would also update the upload section on this page and add the mongodrdl name-schema step: https://www.mongodb.com/docs/bi-connector/current/reference/mongodrdl/#mongodb-commandoption-commandoption.upload
![image](https://github.com/mongodb/docs-bi-connector/assets/91975317/34110e61-ca27-4731-b608-baf22f3642f0)

I think the flow is:
- mongodrdl upload --schemaSource schemas --drdl ./movies.drdl
This returns the id : 5d793f3f6a26a3ce66c304ea
- mongodrdl name-schema --name movies --schemaSource schemas --schema 5d793f3f6a26a3ce66c304ea
- mongosqld --schemaMode custom --schemaSource schemas --schemaName movies

.. include:: /includes/fact-geospatial-views.rst

When you're done editing your ``.drdl`` file:

> 1. Use :binary:`~bin.mongodrdl` to :commandoption:`upload` the schema

Author: Made the updates to this file accordingly. Thank you for the clarification, Kalung!

.. include:: /includes/fact-geospatial-views.rst

When you're done editing your ``.drdl`` file, you can follow the steps in
one of the following options:

- Option 1. Upload the schema, give it a name, and start :binary:`~bin.mongosqld`,
  specifying both the schema's name, source, and mode, as in the following steps:

  1. Use :binary:`~bin.mongodrdl` to :commandoption:`upload` the schema
     to your MongoDB deployment, similar to the following example:

     .. code-block:: sh
   
      mongodrdl upload --schemaSource schemas --drdl ./movies.drdl

  The string representation of the :ref:`ObjectId <document-bson-type-object-id>`
  of the uploaded schema is returned:

  .. code-block:: sh
     :copyable: false
   
      5d793f3f6a26a3ce66c304ea

>   2. Add a name to your schema file, using the :commandoption:`name-schema <name-schema>`

Reviewer: ```suggestion
  2. Name your schema, using the :commandoption:`name-schema <name-schema>`
```

     command, similar to the following example:

  .. code-block:: sh
   
      mongodrdl name-schema --name movies --schemaSource schemas --schema 5d793f3f6a26a3ce66c304ea

>   3. Run :binary:`~bin.mongosqld` with :option:`--schemaSource <mongosqld --schema>`,

Reviewer: ```suggestion
  3. Start :binary:`~bin.mongosqld` with a named schema using :option:`--schemaSource <mongosqld --schema>`,
```

.. include:: /includes/fact-geospatial-views.rst

When you're done editing your ``.drdl`` file, you can follow the steps in
one of the following options:

- Option 1. Upload the schema, give it a name, and start :binary:`~bin.mongosqld`,
  specifying both the schema's name, source, and mode, as in the following steps:

  1. Use :binary:`~bin.mongodrdl` to :commandoption:`upload` the schema
     to your MongoDB deployment, similar to the following example:

     .. code-block:: sh
   

>       mongodrdl upload --schemaSource schemas --drdl ./movies.drdl

Reviewer 2: > mongodrdl upload --schemaSource schemas --drdl ./movies.drdl

```
mongodrdl upload --host <hostName> --schemaSource schemas --drdl ./movies.drdl
```

When you're done editing your ``.drdl`` file, you can either:

- Use :binary:`~bin.mongodrdl` to :commandoption:`upload` the schema
  to your MongoDB deployment. This requires naming your schema file,

>   using the :commandoption:`Name <name-schema>` option.

Reviewer: ```suggestion
  using the :commandoption:`name-schema <name-schema>` command.
```

I think it makes it clearer that the flow is 
1. upload schema
2. name schema
3. Use schema with mongosqld

Author: Updated. Rewrote as:

```
When you're done editing your ``.drdl`` file:

1. Use :binary:`~bin.mongodrdl` to :commandoption:`upload` the schema
  to your MongoDB deployment.
  
2. Add a name to your schema file, using the :commandoption:`Name <name-schema>` option.

3. Run :binary:`~bin.mongosqld` with the
  :option:`--schema <mongosqld --schema>` option to specify the
  ``<my-schema-name>.drdl``.
  
  ```
